ra/ra.go

```diff
@@ -1840,7 +1840,7 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 		// An authz without an expiry is an unexpected internal server event
 		if authz.Expires == nil {
 			return nil, berrors.InternalServerError(
-				"SA.GetAuthorizations returned an authz (%d) with nil expiry",
+				"SA.GetAuthorizations returned an authz (%s) with nil expiry",
```

Comment:
Is this a result of a lint warning? %d is the format verb for base 10 integers, switching to %s seems... wrong?

Reply:
Yes it was from a lint warning. Yes I know what `%d` is :laughing:.

The switch to `%s` seemed correct to me: the argument is `*authz.Id` where `authz` is a `*corepb.Authorization`. The `Id` field of that type is "optional string":
https://github.com/letsencrypt/boulder/blob/43b61f5c253e0d8ffbe7858108b369467eafcbd0/core/proto/core.proto#L58

Reply:
Oh d'oh, yeah that seems right (in my mind id == int).